zoo/entities/yaml_definitions/component_library.yaml

```diff
@@ -0,0 +1,97 @@
+---
+additionalProperties: false
+properties:
+  apiVersion:
+    type: string
+  kind_:
+    enum:
+      - component
+    type: string
+  metadata:
+    properties:
+      description:
+        type:
+          - string
+        nullable: true
+      group:
+        properties:
+          maintainers:
+            items:
+              type: string
+            type: array
+          product_owner:
+            type: string
+          project_owner:
+            type: string
+        type: object
+      links:
+        items:
+          properties:
+            icon:
+              type:
+                - string
+              nullable: true
+            name:
+              type: string
+            url:
+              type: string
+          type: object
+        type:
+          - array
+      name:
+        type: string
+      owner:
+        type: string
+      tags:
+        items:
+          type: string
+        type:
+          - array
+    type: object
+    required:
+      - name
+      - owner
+  spec:
+    properties:
+      analysis:
+        items:
+          properties:
+            exclusions:
+              items:
+                type: string
+              type: array
+          type: object
+        type:
+          - array
+      impact:
+        enum:
+          - profit
+          - customers
+          - employees
+        type: string
+      integrations:
+        items:
+          properties:
+            slack_channel:
+              type: string
+            sonarqube_project:
+              type: string
+          type: object
+        type: object
+      lifecycle:
+        enum:
+          - beta
+          - production
+          - deprecated
+          - discontinued
+        type: string
+      type_:
```

Comment:
and `type_`?

Reply:
is it because of conflicting with `type` on jsonschema? but there shouldn't be any problem with that... am I missing something

and if that's it then there should be better ways to fix it than forcing our users to add random underscores to some fields 😅